yt/odin/requirements.txt

```diff
@@ -3,8 +3,6 @@ Flask==2.0.3
 Werkzeug==2.2.2
 cheroot==10.0.0
 
-# Required by some of the checks
+# Required by some of the checks.
 requests==2.31.0
 tabulate==0.9.0
-
--r ../yt/scripts/python_libraries_requirements.txt
```

Comment:
Why is it removed?

Reply:
Some time ago we were trying to build some working newer build of odin and these requirements had weird compatibility issues. It was not me building it, so I do not know the exact details.

In this new setup, these requirements do not seem to case any trouble (afair I built with them at some point), but frankly I do not see why we need all of them. They take quite a while to install, clog up the image, etc.

Let's put it this way: either a library is necessary for yt client to work properly, then it should be a dependency of the library itself, so that it is auto-installed by pip. If it is somehow optional, let's list such libraries actually needed for odin checks to work. Do we really need numpy, pandas and confluent-kafka for odin checks? 

@savnadya says that a lot of these libraries are required by YT tests, which IMO should be separate from odin